### DISCUSSION TITLE
Plugin for tracking the growth cycle tick for when there's a chance for plants to become diseased

### DISCUSSION THREAD

**Discussion by phage-cell** [runelite/runelite#17472] [Ideas]
https://oldschool.runescape.wiki/w/Farming#Growth_timing

I'd like to minimize my dependency of Geomancy, the spell that lets you see an overview of all your crops. I'd like to be able to track the growth tick cycle with a plugin. Can that be done? A visual aid that shows when the growth tick is coming would be nice.

**Comment by xMistiKmaNx** [comment]
Time tracking plugin, it's included
